react/admin/ReturnDetails/components/StatusHistory.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Table } from 'vtex.styleguide'
+import type { IntlFormatters } from 'react-intl'
+import { FormattedDate, useIntl, FormattedMessage } from 'react-intl'
+
+import { useReturnDetails } from '../../hooks/useReturnDetails'
+import {
+  createStatusTimeline,
+  statusMessageIdAdmin,
+} from '../../../utils/requestStatus'
+
+const statusHistorySchema = (
+  formatMessage: IntlFormatters['formatMessage']
+) => ({
+  properties: {
+    createdAt: {
+      title: (
+        <FormattedMessage id="admin/return-app.return-request-details.table.status-history.header.created-at" />
+      ),
+      cellRenderer: function CreatedAt({ cellData }) {
+        return (
+          <FormattedDate
+            value={cellData}
+            day="numeric"
+            month="long"
+            year="numeric"
+            hour="numeric"
+            minute="numeric"
+            second="numeric"
+          />
+        )
+      },
+    },
+    status: {
+      title: (
+        <FormattedMessage id="admin/return-app.return-request-details.table.status-history.header.status" />
+      ),
+      cellRenderer: function Status({ cellData }) {
+        return formatMessage(statusMessageIdAdmin[cellData])
+      },
+    },
+    submittedBy: {
+      title: (
+        <FormattedMessage id="admin/return-app.return-request-details.table.status-history.header.submitted-by" />
+      ),
+    },
+  },
+})
+
+export const StatusHistory = () => {
+  const { data } = useReturnDetails()
+  const { formatMessage } = useIntl()
+
+  if (!data) return null
+
+  const { status, refundStatusData } = data.returnRequestDetails
+
+  const timeline = createStatusTimeline(status, refundStatusData).filter(
+    ({ visited }) => visited
+  )
+
+  return (
+    <section className="mv4">
+      <Table
```

Comment:
What about adding an `h3` here? 

![Screenshot 2022-06-22 at 11 35 23](https://user-images.githubusercontent.com/50715158/174996693-66063a49-7541-4d4b-ac92-43f0b6073204.png)

Reply:
Fair! Adding it.